package.json

```diff
@@ -4,15 +4,17 @@
   "description": "Angular Sample Applications for Okta",
   "scripts": {
     "test": "npm run test:e2e",
-    "postinstall": "npm install --prefix okta-hosted-login && npm install --prefix custom-login",
     "okta-hosted-login-server": "npm start --prefix okta-hosted-login/",
     "test:okta-hosted-login": "protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
     "custom-login-server": "npm start --prefix custom-login/",
     "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
     "resource-server": "mvn -Dokta.oauth2.issuer=https://${OKTA_DOMAIN}.com/oauth2/default -f samples-java-spring-mvc/resource-server/pom.xml",
-    "test:e2e": "export TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
+    "test:e2e": "cross-env TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
     "get-oidc-tck": "[ ! -d okta-oidc-tck/ ] && git clone https://github.com/okta/okta-oidc-tck.git || echo \"TCK already cloned\"",
-    "pretest": "webdriver-manager update --gecko false"
+    "pretest": "webdriver-manager update --gecko false",
+    "postinstall": "npm run install-custom-login && npm run install-okta-hosted-login",
```

Comment:
On the second call here, do we need to `cd` back into our working directory?

Reply:
Running `npm install` in the root doesn't actually `cd` into the sub-directory on the console. I think we are good here. I've testing `npm install` on mac and windows. Both look fine.